capsules/src/net/network_capabilities.rs

```diff
@@ -48,7 +48,7 @@ impl AddrRange {
                 if &allowed_addr.0[0..full_bytes] != &addr.0[0..full_bytes] {
                     false
                 } else {
-                    allowed_addr.0[full_bytes] >> (8 - remainder_bits)
+                    addr.0[full_bytes] >> (8 - remainder_bits)
```

Comment:
This does not look like a clippy fix, that's reading from a different variable?

Reply:
clippy pointed out that both sides of the comparison were identical. I looked into this and realized that was a logic error! In fact we want to be comparing the allowed address subnet to the passed address subnet and see if they match.

(clippy did not autofix this, I looked at all of the clippy warnings and manually patched them)

Reply:
(I can move this to another PR if preferred, which could be included in 1.5 if desired?)

Reply:
If it's not too onerous, I think it'd be good to pull that out into its own fix. Lumping a real bug / logic fix into a big pile of formatting fixes risks it getting lost.

Re 1.5: I think you're responsible for all of the testing on the boards that exercise the networking code path, so if you're willing to re-test / own the change, then yeah, getting the bugfix in is probably good.